lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -533,6 +533,37 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
   enqueueKernel(commands_, kernel, deviceId_, global, local, kernelLaunches_);
 }
 
+template <typename T>
```

Comment:
Is it copied from e.g. interpreter backend? If so, please mention it in the comment. So that we know they should be in sync.

Reply:
done